WooCommerce/Classes/ViewRelated/Feature Highlight/SpotlightView/QuickStartSpotlightView.swift

```diff
@@ -0,0 +1,214 @@
+import UIKit
+
+class QuickStartSpotlightView: UIView {
```

Comment:
💭 since you mentioned that we aren't using `QuickStartSpotlightView`, can this be removed from the PR? Otherwise, I'll take some time to review this if it's being merged to the code base.

Reply:
Removed this in https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/b375194df51fa64127dbd7194dbb8ba155b54c13